dir.props

```diff
@@ -152,6 +152,13 @@
 
     <ProjectUrl>https://dot.net</ProjectUrl>
 
+    <!-- Central place to set the versions of all nuget packages produced in the repo -->
+    <Version Condition="'$(Version)' == ''">1.1.0</Version>
```

Comment:
@ericstj do you mean usage of Version in CoreCLR repo or in external components like BuildTools?